robberies/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Informații generale
+---
+
+### Jafuri disponibile:
+
+[HotDog Stand Robbery](./hotdog-robbery) - Nivel 1-2, necesită o armă
+[ATM Robbery](./atm-robbery) - Nivel 3-15, necesită vehicul personal, obiect craftat pe insula Cayo Perico (Hooked Rope)
+[Fleeca Bank Robbery](./fleeca-bank-robbery.md) - Nivel 15+, necesită o armă cu muniție, obiect craftat pe insula Cayo Perico (C4 Bomb), obiecte adiționale (Drill, Pliers)
+
+### Care sunt skill-urile la Robbing (Street Reputation)?
+
+| Numele skill-ului | Puncte necesare pentru avansare | Bonus $ |
+| :-----------: | :-----------: | :-----------: |
+| Prospect | **0** |  _**0%**_ |
+| Snitch | **100** |  _**5%**_ |
+| Shoplifter | **250** |  _**15%**_ |
+| Crook | **750** |  _**20%**_ |
+| Burglar | **1500** |  _**25%**_ |
+| Thug | **3000** |  _**30%**_ |
+| Capo | **6000** |  _**35%**_ |
+| Consigliere | **12000** |  _**40%**_ |
+| Don | **24000** |  _**50%**_ |
+| Kingpin | **48000** |  _**60%**_ |
+| Padrino | **96000** |  _**70%**_ |
+| Godfather | **192000** |  _**80%**_ |
+
+### Ce este Heat Level?
+
+Heat Level-ul este un sistem de delay (amânare) al activităților ilegale, ce te împiedica din practicarea acestora deoarece atragi prea multă atenție.
+
+De fiecare dată când vei jefui un ATM, vei primi 100% Heat Level, care va scădea complet în 5 ore (1% per 3 minute).
+
+| Heat Level | Cât durează să dispară? | Bonus, mai rapid cu 15%<br>**<Color hex="#FFD700">Gold Subscription</Color>** | Bonus, mai rapid cu 30%<br>**<Color hex="#100C84">Platinum Subscription</Color>** |
+| :-----------: | :-----------: | :-----------: | :-----------: | 
+| 100% | 5 ore | 4 ore și 15 minute | 3 ore și 30 minute |
+| 75% | 3 ore și 45 minute | 3 ore, 11 minute și 15 secunde | 2 ore și 45 minute |
+| 50% | 2 ore și 30 minute | 2 ore, 7 minute și 30 secunde | 1 oră și 45 minute | 
+| 25% | 1 oră și 15 minute | 1 oră, 3 minute și 45 secunde | 52 minute și 30 secunde |
+| 10% | 30 minute | 25 minute și 30 secunde | 21 minute |
+| 1% | 3 minute | 2 minute și 33 secunde | 2 minute și 6 secunde |
+
+# Cayo Perico
+
+Pe server a fost adaugata insula **Cayo Perico**. Insula este disponibila pe minimap si harta din ESC tot timpul. Insula dispune de mapping mapat pentru toate interioarele de pe aceasta.
+
+Pe **Cayo Perico** se pot desfasura urmatoarele activitati:
+- Spalarea de bani: Aceasta activitate poate fi desfasurata doar de catre jucatorii aflati intr-un clan ce are deblocat perk-ul "Money Laundering".
+- Craftare iteme necesare Bank & ATM Robbery: Jucatorii pot crafta itemele necesare celor doua jafuri
+- Craftare bijuterii: Jucatorii pot crafta bijuterii
+- Port: Jucatorii pot spawna barcile personale
+- Loc vanzare pesti: Jucatorii pot vinde pestii pescuiti la jobul **Deep Sea Fishing**
+
+## Pawn Shop
+
```

Comment:
**Ce pot face cu "Gold Bar" & "Silver Bar"?**

Cu "Gold Bar" & "Silver Bar" iti poti crafta anumite articole vestimentare pe insula "Cayo Perico", sau le poti vinde in amanet. Pentru a localiza foloseste comanda [/gps] -> Businesses -> Pawn Shop.